07-regression1.Rmd

```diff
@@ -500,7 +527,7 @@ that we are trying to predict.
 It is always a good practice to do exploratory data analysis, such as
 visualizing the data, before we start modeling the data. Thus the first thing
 we will do is use ggpairs (from the `GGally` package) to plot all the variables
-we are interested in using in our analyses:
+we are interested in using in our analyses.
 
 ```{r 07-ggpairs, fig.height = 5, fig.width = 6, fig.cap = "Scatterplots of each pair of variables (price, house size, and the number of bedrooms) are displayed in the figure's bottom left corner. Correlation coefficients are shown in the top right corner. The distributions of price, house size and number of bedrooms are each shown along the diagonal."}
```

Comment:
Since we only look at price vs sqft and price vs bedrooms - I wonder if we just want to make two scatter plots? That would be far simpler for students to understand (especially since we have never talked about densities or correlations before...).

Also, and MDS student reading this chapter this year, was concerned in the linear regression chapter with beds and sqft being so highly correlated, so that there might be a multicollinearity issue... I admit I didn't think about that before and probably should have... It is something we might want to address somehow.

Reply:
- agree that we should just put a scatter of price vs beds
- re: multicollinearity. If you look at the 3d scatter, you can see that there is reasonable signal on the full direction of the plane. shouldn't be a big problem... maybe worth some more thought